cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md

```diff
@@ -341,6 +341,7 @@ The `SanitizeHelper` module provides a set of methods for scrubbing text of unde
 **Other libraries that provide HTML Sanitization include:**
 - [HTML sanitizer](https://github.com/google/closure-library/blob/master/closure/goog/html/sanitizer/htmlsanitizer.js) from [Google Closure Library](https://developers.google.com/closure/library/) (JavaScript/Node.js, [docs](https://google.github.io/closure-library/api/goog.html.sanitizer.HtmlSanitizer.html))
 - [DOMPurify](https://github.com/cure53/DOMPurify) (JavaScript, requires [jsdom](https://github.com/jsdom/jsdom) for Node.js)
+- [sanitize-html](https://github.com/apostrophecms/sanitize-html) (JavaScript/Node.js)
```

Comment:
Are you sure that this library in default configuration protects against XSS? I did not used it but from a quick look into documentation security/preventing XSS is not the main purpose of this library. In compartion DOMPurify is for it, is well tested, and is maintained by great frontend security professionals. 
I am not sure about adding third JavaScript sanitizer, and for sure before adding it I would have to check if this lib really protects from XSSes.

Reply:
Good point! I'll remove it for this PR